Admin/LifecycleHookProviderInterface.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+/*
+ * This file is part of the Sonata Project package.
+ *
+ * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sonata\AdminBundle\Admin;
+
+/**
+ * This interface can be implemented to provide hooks that will be called
+ * during the lifecycle of the object.
+ */
+interface LifecycleHookProviderInterface
+{
+    /**
+     * @param object $object
```

Comment:
I know that you just moved the methods. But can you find useful descriptions for the methods?